### PR TITLE
Fix #239: Added backend option details

### LINKED ISSUE #239: Multi-Array Support: -o backend option not available in "help"
-o backend=[backend name]  option is not available in "help"

Created a pull request and added the above -o backend=[backend name]  option in help.

### PR DESCRIPTION
**Pull request by sandesh-desai**

### CODE REVIEW

File: config/create_help.txt [hunk 1]
```diff
@@ -31,6 +31,8 @@ Import Volume Options:
  -o importVol=x                 x is the name of 3PAR volume or snapshot which needs to be imported. Volume or snapshot which needs to be imported
                                 should not be attached to any of the host.
 
+Backend Volume Options:
+ -o backend=x                   x is the backend name on which the operation to be performed.
```
Comment (wdurairaj) [inline config/create_help.txt]: Add a line like

```
Refer https://github.com/hpe-storage/python-hpedockerplugin/blob/plugin_v2/docs/multi-array-feature.md for details.
```

File: config/create_help.txt [hunk 2]
```diff
@@ -31,6 +31,8 @@ Import Volume Options:
...
```
Comment (nilangekarss) [inline config/create_help.txt]: This line has to be added under create volume options field as well

Comment (nilangekarss) [reply config/create_help.txt]: @sandesh-desai 
please remove 'Backend Volume Options'
simply keep -o backend=x and description under Import volume options

**Comment by wdurairaj** [comment]
This is taken care via pull request 
https://github.com/hpe-storage/python-hpedockerplugin/pull/328/files

**Pull request closed without merging**